readthedocs/oauth/management/commands/sync_vcs_data.py

```diff
@@ -60,6 +78,11 @@ def handle(self, *args, **options):
             socialaccount__isnull=False
         ).distinct()
 
+        if logged_in_days_ago:
```

Comment:
```suggestion
        if logged_in_days_ago > 0:
```

Maybe we should check `if logged_in_days_ago > 0` ? As we have a default for this argument, and if it's `0` or Not provided, it will just filter `last_login__gte=timezone.now()` which I think we don't want. :)

Reply:
Well, if it's 0 (or not provided) it will do `if 0:` and won't enter to the if.

I'm fine adding it explicitly, tho.

Reply:
Oops! My bad :)